haiku-apps/catkeyseditor/catkeyseditor-0.1.1.recipe

```diff
@@ -10,7 +10,10 @@ HOMEPAGE="https://github.com/puckipedia/BeLocalized"
 COPYRIGHT="2014-2015 Puck Meerburg"
 LICENSE="MIT"
 REVISION="1"
```

Comment:
> Sorry @dash102, I just noticed that I should have asked you to bump REVISION.
This is required because we switched from 989904ff0181147e5fe7e205096f09e1cc4d8bdd to ec3e5accedb2b891abba96a94ff3293f7eca60f6.
Thanks again!

Hmm, I don't know if we should use a newer srcGitRev. I guess we could, but I'd recommend that we wait for a go from a project leader.
We can either:
* limit this edit to a neutral change:
```
REVISION="1"
srcGitRev="989904ff0181147e5fe7e205096f09e1cc4d8bdd"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="894eab6f25b6f67afadea018b4ab9508af927cc7ddd349b4a924bc3ba832b6f6"
```
* or bump to the newer gitSrcRev:
```
REVISION="2"
srcGitRev="ec3e5accedb2b891abba96a94ff3293f7eca60f6"
SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
CHECKSUM_SHA256="39922377cb3207077a57e9887bbbf1fbb190c099aac6110acd61f31dddb5b068"
```
(in which case you just need to bump REVISION.)

Well, let's see what the other contributors say about this.